flink-python/pyflink/table/table_config.py

```diff
@@ -239,7 +239,7 @@ def get_min_idle_state_retention_time(self) -> int:
 
         :return: The minimum time until state which was not updated will be retained.
         """
-        return self._j_table_config.getMinIdleStateRetentionTime()
+        return self._j_table_config.getIdleStateRetention().toMillis()
```

Comment:
I think these python methods `get_min_idle_state_retention_time` and `get_max_idle_state_retention_time` can also be removed because they have been tagged `deprecated` in comments. cc @HuangXingBo can you help to double check it?

Reply:
done :)